fiftyone/utils/cvat.py

```diff
@@ -2377,6 +2425,2066 @@ def write(
         etau.write_file(xml_str, xml_path)
 
 
+class CVATAnnotationAPI(foua.BaseAnnotationAPI):
+    """A class to facilitate connection to and management of tasks in CVAT.
+
+    On initializiation, this class constructs a session based on the provided
+    server url and credentials.
+
+    This API provides methods to easily get, put, post, patch, and delete tasks
+    and jobs through the formatted urls specified by the CVAT REST API.
+
+    Additionally, samples and label schemas can be uploaded and annotations
+    downloaded through this class.
+
+    Args:
+        url ("cvat.org"): URL of the CVAT server to which to upload samples
```

Comment:
How about deleting the `https` parameter and just asking the user to put `https://cvat.org`, `http://my-cvat.com` or `localhost` or whatever into the `url` parameter? Fewer parameters = better.

You can deduce/strip out the appropriate `http/s` value if you really need it.